### DISCUSSION TITLE
deny editing for specific acl groups.

### DISCUSSION THREAD

**Discussion by AndiLeni** [joomengine/Joomla-Component-Builder#802] [Q&A] [answered]
Hello,
I want to know wether the usecase I have in mind can be made possible. 

I have a admin view where editors can create items. 
What I want is the ability to restrict editing items for specific departments. 
F.e. one person of department A creates a few items, all members of department A should be able to edit these items, but users of the departments B, C and so on should not be able to edit them. 

Kind regards
Andreas

**Comment by ohrionmartin** [comment, marked as answer]
Hi @AndiLeni 

Check out this [video](https://youtu.be/CdSKSCTzmRA?t=379) for the admin area and this [one](https://youtu.be/gWjQjdhYqXI) for the site area, hope it helps..
